modules/ROOT/pages/supported-cluster-config.adoc

```diff
@@ -25,26 +25,26 @@ The following recommendations and limits are based on these resources:
 
 == Mule Runtime Engine and Mule Applications
 
-The following table shows the recommended and maximum number of Mule instances and Mule applications that Anypoint Platform PCE supports:
+This  table shows the recommended and maximum number of Mule instances and Mule applications that Anypoint Platform PCE supports:
 
 [%header%autowidth.spread]
 |===
-| Description| Small cluster 4-node Configuration | Large cluster 7-node Configuration
-| Maximum number of Mule instances per environment | 500 | 500
-| Maximum number of Mule instances across all environments | 1500 | 3000
-| Maximum number of Mule instances per server group | 20 | 20
-| Maximum Mule instances per cluster | 8 | 8
-| Mule applications under management per environment (recommended) | Under 200 | Around 200
-| Mule applications under management per environment (maximum) | 500 | 500
-| Mule applications per Mule runtime engine (recommended) | Fewer than 25 | Fewer than 25
-| Mule applications per Mule runtime engine (maximum) | 100 | 150
-| Number of Mule applications | 6000 | 12000
-| Number of Mule application replicas. An application can run on multiple Mule instances in a server group or cluster. | 12000| 24000
+| Description| Supported Capacity
+| Maximum number of Mule instances per environment | 500 
+| Maximum number of Mule instances across all environments | 4000
+| Maximum number of Mule instances per server group | 20 
+| Maximum Mule instances per cluster | 8 
+| Mule applications under management per environment (recommended) | Under 200 
+| Mule applications under management per environment (maximum) | 500 
+| Mule applications per Mule runtime engine (recommended) | Fewer than 25 
+| Mule applications per Mule runtime engine (maximum) | 100 
+| Number of Mule applications | 15000
+| Number of Mule application replicas. An application can run on multiple Mule instances in a server group or cluster. | 30000
 |===
 
 == APIs and API Manager
```

Comment:
The sections needs an update. All the sections in this page are updated in the G-Doc. Please refer the same.

Reply:
Added! :)